### PR TITLE
WIP: onboard Serverless for Z-Stream testings

### PR DESCRIPTION
**Pull request by liswang89**
Z-Stream testing on Serverless 1.31 release.

### CODE REVIEW

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.30-ocp4.13-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.30-ocp4.13-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.30-ocp4.14-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.13-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.13-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

**Comment by maschmid** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

let's see if https://github.com/openshift-knative/eventing-kafka-broker/pull/932 made it better

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.30__ocp4.14-lp-interop.yaml [hunk 1]
```diff
@@ -99,6 +99,80 @@ resources:
...
+              {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster"},
+              {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators"},
+              {"step": "gather-*", "failure_type": "all", "classification": "Other"},
+              {"step": "openshift-*", "failure_type": "all", "classification": "Other"},
+              {"step": "knative-eventing-kafka-broker-e2e", "failure_type": "test_failure", "classification": "knative-eventing-kafka-broker-e2e Test Failure", "jira_project": "SRVCOM"},
+              {"step": "knative-serving-eventing-e2e", "failure_type": "test_failure", "classification": "knative-serving-eventing-e2e Test Failure", "jira_project": "SRVCOM"},
+              {"step": "operator-e2e", "failure_type": "test_failure", "classification": "operator-e2e Test Failure", "jira_project": "SRVCOM"}
+            ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.14-lp","self-managed-lp","serverless-lp"]'
```
Comment (Simranpal) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.30__ocp4.14-lp-interop.yaml:121]: We can add the z-stream label as well like Adam did here:
https://github.com/openshift/release/pull/46901/files#diff-dcc1c6f6c814d60c4e9402d430743cfb190f7a8fdcb1aaab05a3bee3ecb533ffR56

**Review by Simranpal — COMMENTED** [review]
1) I see there are two product versions tested for ocp4.14x, is this an exception?
2) I am wondering where to distinguish the z-stream tests from self-managed, seperate files vs seperate test stanza name or just labels.

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.14-lp-interop-operator-e2e-interop

**Comment by liswang89** [comment]
/pj-rehearse pull-ci-openshift-knative-serverless-operator-release-1.31-ocp4.13-lp-interop-operator-e2e-interop

**Pull request closed without merging**